src/features/pastelPaperWalletGenerator/pastelPaperWalletGenerator.tsx

```diff
@@ -0,0 +1,192 @@
+import {
+  Document,
+  Page,
+  Path,
+  PDFViewer,
+  StyleSheet,
+  Svg,
+  Text,
+  View,
+} from '@react-pdf/renderer'
+import QRCode from 'qrcode.react'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import Modal from 'react-modal'
+
+import styles from './PastelPaperWalletGenerator.module.css'
+
+type PastelPaperWalletGeneratorProps = {
+  address: string
+  privateKey: string
+  modalIsOpen: boolean
+  onCloseModal: () => void
+}
+
+function PastelPaperWalletGenerator(
```

Comment:
Strictly speaking, this is a `PastelPaperWalletModal`, and the `generateQrcode` function is actually a `QRCodeGenerator`. You can change the component names as you like, I just wanted to give an idea how they should look like

Reply:
@br4in3x yeah i have changed the component names as your suggest